src/Components/Dropdowns/Composed/TypeAheadDropDown.tsx

```diff
@@ -70,16 +48,10 @@ export const TypeAheadDropDown: FC<OwnProps> = ({
   items,
   onSelect,
   testID,
-  placeholderText,
-  testIdSuffix = 'typeAhead',
+  placeholderText = 'Select a Value',
   selectedOption = null,
   className,
-  largeListSearch = false,
-  largeListCeiling = 0,
   menuTheme = DropdownMenuTheme.Onyx,
-  buttonTestId = 'type-ahead-dropdown--button',
-  menuTestID = 'type-ahead-dropdown--menu',
-  itemTestIdPrefix = 'type-ahead-dropdown--item',
```

Comment:
less configurable options to make test-ids useable in the UI. Spoke with @hoorayimhelping about the reasoning behind this change.

TLDR: we didn't have a specific need for this ultra-configurable test id setup, if we need it in the future we can consider adding it. At the moment this was just making it hard to use it.